README.md

```diff
@@ -291,6 +296,125 @@ class YourClass {
+
+The `DefaultExemplarSampler` comes with built-in support for [OpenTelemetry tracing](https://github.com/open-telemetry/opentelemetry-java), see  [built-in support for tracing systems](#built-in-support-for-tracing-systems) below.
+
+You can disable the default exemplar sampler globally with:
+
+```java
+ExemplarConfig.disableExemplarSamplers();
+```
+
+You can set your own custom implementation of `ExemplarSampler` as a global default like this:
+
+```java
+ExemplarSampler myExemplarSampler = new MyExemplarSampler();
+ExemplarConfig.setCounterExemplarSampler(myExemplarSampler);
+ExemplarConfig.setHistogramExemplarSampler(myExemplarSampler);
+```
+
+Note that `ExemplarConfig` is used when a metric is created, so changing the default has no effect on metrics that have already been created.
+
+### Per Metric Exemplar Samplers
+
+The metric builders for `Counter` and `Histogram` have methods for setting the exemplar sampler for that individual metric. This takes precedence over the global setting in `ExemplarConfig`.
+
+The following calls enable the default exemplar sampler for individual metrics. This is useful if you disabled the exemplar sampler globally with `ExemplarConfig.disableExemplarSamplers()`.
+
+```java
+Counter myCounter = Counter.build()
+    .name("number_of_events_total")
+    .help("help")
+    .withExemplarSampler()
+    ...
+    .register();
+```
+
+```java
+Histogram myHistogram = Histogram.build()
+    .name("my_latency")
+    .help("help")
+    .withExemplarSampler()
+    ...
+    .register();
+```
+
+The following calls disable the default exemplar sampler for individual metrics:
+
+```java
+Counter myCounter = Counter.build()
+    .name("number_of_events_total")
+    .help("help")
+    .withoutExemplarSampler()
+    ...
+    .register();
+```
+
+```java
+Histogram myHistogram = Histogram.build()
+    .name("my_latency")
+    .help("help")
+    .withoutExemplarSampler()
+    ...
+    .register();
+```
+
+The following calls set a custom `MyExemplarSampler` for individual metrics:
+
+```java
+// CounterExemplarSampler
+Counter myCounter = Counter.build()
+    .name("number_of_events_total")
+    .help("help")
+    .withExemplarSampler(new MyExemplarSampler())
+    ...
+    .register();
+```
+
+```java
+// HistogramExemplarSampler
+Histogram myHistogram = Histogram.build()
+    .name("my_latency")
+    .help("help")
+    .withExemplarSampler(new MyExemplarSampler())
+    ...
+    .register();
+```
+
+### Per Observation Exemplars
+
+You can explicitly create an exemplar for an individual observation. This takes precedence over the exemplar sampler configured with the metric.
+
+The following call will increment a counter, and create an exemplar with the specified `span_id` and `trace_id` labels:
+
+```java
+myCounter.incWithExemplar(Exemplar.SPAN_ID, "abcdef", Exemplar.TRACE_ID, "123456");
```

Comment:
I'd include the literal string here, rather than pointing off to an unspecified class that you have to import to get to a constant.

Reply:
Removed the `TRACE_ID` String from `Exemplar`.

Reply:
Did you forget to push? I'm not seeing a change here.